README.md

```diff
@@ -1,10 +1,9 @@
 # phimpme-android
-<img src="/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png" align="left" width="200" hspace="10" vspace="10">
+![Phimpme](/docs/_static/logo.png)
```

Comment:
@imujjwal96 This one. The image doesn't need to be inside `docs/_static`, but it does need to be inside `docs`